docs/deploying/deploy-faucets.mdx

```diff
@@ -0,0 +1,30 @@
+---
+sidebar_position: 3
+toc_max_heading_level: 5
+---
+
+# Faucets (Available)
+
+Faucets for deployment of smart contracts can be requested from the website.
+
+The button is at the bottom left of the  [Demo app] (https://scaffold-stark-demo.vercel.app/debug)
```

Comment:
This hyperlink doesn't show what you've intended I think. Can check the syntax again.

<img width="661" alt="image" src="https://github.com/user-attachments/assets/0b413575-e0cb-428d-80da-fe35c3969148">

Reply:
@pheobeayo This is not resolved yet, you can try editing while running the localhost instance with `yarn start`. The vscode markdown doesn't necessarily give you the right representation.